src/custom/state/claim/reducer.ts

```diff
@@ -53,7 +53,7 @@ export type ClaimState = {
   isSearchUsed: boolean
   // claiming
   claimStatus: ClaimStatus
-  claimedAmount: number
+  claimedAmount: string
```

Comment:
this is also derived from the state, although that was already the case before this PR. I guess we replicate for convenience, right?

Reply:
It's a bit expensive/annoying to calculate.
The state has the amounts invested in the invested token, so we need to build the currencyamount objects and apply the prices to calculate the vCow amounts and them sum it up.

Reply:
okok